rules-tests/CodeQuality/Rector/Class_/AddCoversClassAttributeRector/config/configured_rule.php

```diff
@@ -4,7 +4,10 @@
 
 use Rector\Config\RectorConfig;
 use Rector\PHPUnit\CodeQuality\Rector\Class_\AddCoversClassAttributeRector;
+use Rector\PHPUnit\ValueObject\TestClassSuffixesConfig;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->rule(AddCoversClassAttributeRector::class);
+    $rectorConfig->ruleWithConfiguration(AddCoversClassAttributeRector::class, [
```

Comment:
please create separate config files, fixture dir, and test class to ensure existing functionality still works